src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -0,0 +1,159 @@
+
+Below are two examples of custom queries.
+
+<CollapserGroup>
+    <Collapser
+        className="freq-link"
+        id="longest"
+        title="Longest queries"
+    >
+    To find the queries that take the longest to load:
+
+    1. Create a file and name it `longest_queries.sql` 
+    2. Designate the file path as `newrelic-snowflake-integrations/queries`.
+    3. In `longest_queries.sql`, add the following snippet:
+
+    ```sql
+    select 
+        query_id, 
+        query_text,
+        (execution_time / 60000) as exec_time,
+        warehouse_name,
+        user_name,
+        execution_status
+    from 
+        snowflake.account_usage.query_history 
+    where 
+        execution_status = 'SUCCESS' 
+    order by 
+        execution_time desc
+    ```
+    4. In the `flex-snowflake-linux.yml` file, add the following snippet:
+    
+```yml
+- name: longestQueries
+entity: snowflake
+# New Relic will capture all your Snowflake metrics when you use `event_type: SnowflakeVirtualWarehouse`. 
+event_type: SnowflakeVirtualWarehouse
+custom_attributes:
+    metric_type: snowflake.query_performance
+commands: 
+    - run: - run: <var>YOUR_PATH_TO_DOWNLOADED_BINARY_FILE</var> <var> YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/config.yaml</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/queries/longest_queries.sql</var>
+```
+    </Collapser>
+    <Collapser
+        className="freq-link"
+        id="warehouse-usage"
+        title="Credit usage by warehouse"
+    >
+    To find the queries that take the longest to load:
+
+    1. Create a file and name it `credit_usage_by_warehouse.sql` 
+    2. Designate the file path as `<var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY</var>/queries/credit_usage_by_warehouse.sql`.
+    3. In `credit_usage_by_warehouse.sql`, add the following snippet:
+
+    ```sql
+    select 
+        warehouse_name, 
+        sum(credits_used) as total_credits_used 
+    from 
+        snowflake.account_usage.warehouse_metering_history 
+    group by 
+        1 
+    order by 
+        2 desc;
+    ```
+    4. Add the following snippet to `flex-snowflake.yml`:
+
+    ```yml
+    - name: CreditUsageByWarehouse
+        entity: snowflake
+        event_type: SnowflakeVirtualWarehouse
+        custom_attributes:
+            metric_type: snowflake.credit_usage_by_warehouse
+        commands: 
+        - run: <var>YOUR_PATH_TO_DOWNLOADED_BINARY_FILE</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/config.yaml</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/credit_usage_by_warehouse.sql</var>
+    ``` 
+    </Collapser>
+</CollapserGroup>
+
+## Get Snowflake metrics in a dashboard [#dash]
+
+Once you've installed the [Snowflake quickstart](https://newrelic.com/instant-observability/snowflake), you can see your critical Snowflake data in New Relic. 
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then click **Metrics & events**.
```

Comment:
we could maybe be more clear that these are instructions for after they install the quickstart and want to see the dashboards. at first I thought these were instructions for installing the quickstart.